.github/workflows/c-cpp.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        os: [macos-latest, ubuntu-latest, windows-2022]
```

Comment:
Something tells me this should not be changed.

Reply:
Is obligatory too

Reply:
> Something tells me this should not be changed.

Look at the specification of included packages for the GitHub Actions [`windows-latest` // `windows-2019`](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#visual-studio-enterprise-2019) and [`windows-2022`](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md#visual-studio-enterprise-2022) virtual environments. `windows-2022` it's the only one set up with Visual Studio 2022.

Reply:
Yeah cool. I guess that's more a GitHub naming issue. Just saw the detail in https://github.com/actions/virtual-environments/issues/4488 so all good.